### PR TITLE
add note to avoid Promise already satisfied

### PR DESCRIPTION
**Pull request by Katawann**
related to issue [https://github.com/mavlink/MAVSDK/issues/996](https://github.com/mavlink/MAVSDK/issues/996)

### CODE REVIEW

File: en/guide/telemetry.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ Similarly, you can use the asynchronous method and block
     // Now ready to arm
 }
 ```
+> **Note** Make sure to unsubscribe from the callback after it suceed using `telemetry->health_all_ok_async(nullptr)`. Otherwise you will run into an error because `prom->set_value()` is called more than once.
```
Comment (hamishwillee) [inline en/guide/telemetry.md]: spelling is "succeed".

Comment (hamishwillee) [reply en/guide/telemetry.md]: @JonasVautherin Anything to add to this?

Comment (Katawann) [reply en/guide/telemetry.md]: Don't need to add it in the code example IMO. The example is working while it is not called more than once

Comment (Katawann) [reply en/guide/telemetry.md]: I need to do a new commit to propose your new note ?

Comment (hamishwillee) [reply en/guide/telemetry.md]: @Katawann You would have but I did it for you now (I would have done this first time, but I wanted to make sure it still addresses your concern).

@JonasVautherin @julianoes Can either of you confirm that you're now happy with this PR?

**Comment by hamishwillee** [comment]
Merged! Thanks very much @Katawann !

**Pull request merged**